packages/esm-patient-forms-app/src/forms/forms-dashboard.component.tsx

```diff
@@ -21,6 +21,10 @@ const FormsDashboard: React.FC<DefaultWorkspaceProps> = () => {
   const { patient, patientUuid } = usePatient();
   const { data: forms, error, mutateForms } = useForms(patientUuid, undefined, undefined, !isOnline, config.orderBy);
   const { currentVisit } = useVisitOrOfflineVisit(patientUuid);
+  const sessionUser = useSession();
+  const sessionPrivileges = sessionUser?.user?.privileges;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  let newForms = [];
```

Comment:
Minor issue with the naming here because it's a little misleading. The forms aren't new but rather a filtered version based on privileges. Maybe `filteredForms` could be an option but not sure if that is the best description. 

Am sure @denniskigen has a bot for naming 🤖

Reply:
Or possibly `editableForms`.

Reply:
@ibacher @pirupius changed it 
Thanks